dataset/dictionary/characters.json5

```diff
@@ -784,6 +784,14 @@
     notes: "v4.1 期間限定イベント「流れゆく水に詩を紡いで」に登場する純水精霊",
     notesZh: "v4.1 活动「游水酝诗籍」的纯水精灵",
   },
+  {
+    en: "Sir Pouncelot",
+    ja: "橙色騎士",
+    zhCN: "大桔骑士",
+    pronunciationJa: "おれんじきし",
+    tags: [ "mondstadt", "character-sub" ],
```

Comment:
Can you add `"event"` tag and move under events.json5?
I add the event-only characters in event.json5.